rust/operator-binary/src/main.rs

```diff
@@ -79,6 +92,14 @@ async fn main() -> anyhow::Result<()> {
             )
             .await?;
 
+            let event_recorder = Arc::new(Recorder::new(
```

Comment:
We need a different event_recorder for every controller. This is because the controller name is in the recorder.
Please have a look at Superset as the template

Reply:
good catch thanks https://github.com/stackabletech/spark-k8s-operator/pull/515/commits/64172e402d790cd4e641a0fe92c1637e866b0f93

Reply:
Thanks!